### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Create New Web App_Keyboard: Tooltip given for the error icon is not available to the keyboard users

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3337]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.1.1

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select New JSP File (html) and press Finish
8) Open JSP file and write any code in the editor
9) In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App
10) Tab till Create button and press enter
11) Give any wrong details and tab till ‘Create’ button
12) Verify the tooltip given for the error icon is available to the keyboard users

**Actual Result:** 
Tooltip given for the error icon is not available to the keyboard users

**Expected Result:** 
Tooltip given for the error icon must be available to the keyboard users

**User Impact:** 
Keyboard users will not know what the error is if tooltip is not visible for the users

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

![Bug_15_Tooltip given for the error message is not available to keyboard users](https://user-images.githubusercontent.com/41109135/62205446-14baa080-b3ad-11e9-8573-c575f274726a.png)

**Comment by Flanker32** [comment]
@Tanushagottipati This has been fixed in https://github.com/microsoft/azure-tools-for-java/pull/3488, and here is the link for [build](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_PR/3749/Azure/processDownloadRequest/artifacts/WindowsAzurePlugin4EJ.PR.3749.08-30-2019.zip), could you please help us verify it, thanks!

**Comment by Tanushagottipati** [comment]
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
Issue fixed by downloading above build. hence closing the bug and attached latest video for reference
![Now, tooltip is coming at the top](https://user-images.githubusercontent.com/41109135/64530644-48cd9d80-d32b-11e9-98d1-3e56a7adcf2d.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749802
